src/views/CampusSafety/views/LostAndFound/index.tsx

```diff
@@ -169,34 +167,47 @@ const LostAndFound = () => {
             />
             <CardContent>
               <Grid container alignItems="center">
-                <Grid item xs={12} sm={0.5}>
-                  <InfoOutlinedIcon />
-                </Grid>
                 <Grid item xs={12} sm={11}>
                   <Typography variant="body1" align="center">
-                    Police staff will view reports, and you will be notified if your item is found.
+                    <InfoOutlined /> Police staff will view reports, and you will be notified if
+                    your item is found.
                   </Typography>
+                  {/* Button to Report a Missing Item */}
+                  <Grid
+                    container
+                    justifyContent="center"
```

Comment:
The grids don't seem to be properly spaced on the page, so this line doesn't work as intended (at least when I run it). 
<img width="1728" alt="Screenshot 2024-11-29 at 10 08 46 PM" src="https://github.com/user-attachments/assets/a5e9a6bb-741e-46e0-b921-dff95d45fa19">

Reply:
I just pushed a fix to UI #2442 which fixes this :+1: